README.md

```diff
@@ -5,12 +5,22 @@ eslint config rules for Particle Javascript projects
 
 1. Install dependencies
 `npm install --save-dev eslint eslint-config-particle`
```

Comment:
Does the application also need a specific version of the `eslint` package? Or should `eslint` be removed?

Reply:
`eslint` still needs to be installed so we get the binary, should be 9.x at least.